### ISSUE TITLE
Release settings should be removed from the Android build.gradle

### ISSUE THREAD

**Issue by miguelpruivo** [FogosPT/fogosmobile#22]
Because any cloners won't release the app and since the`keystore` file isn't available, these lines:
```
  signingConfigs {
        release {
            keyAlias keystoreProperties['keyAlias']
            keyPassword keystoreProperties['keyPassword']
            storeFile file(keystoreProperties['storeFile'])
            storePassword keystoreProperties['storePassword']
        }
    }
```
as well as this one:
```
signingConfig signingConfigs.release
```
should be removed from the `android/app/src/build.gradle`

This has some relevance because builds will fail with this uncommented/removed.

**Comment by guivazcabral** [comment]
I'm not sure, but @Vanethos can clarify things for me:

Aren't this configs needed for codemagic to release the app _automagically_ ?

**Comment by miguelpruivo** [comment]
@guivazcabral yes, they might be. However it needs to have the keystore file along with it or the build will fail.

I believe that the keystore is being added before releasing so this should be added in that step as well, since we are committing on the `master` directly. 

In my fork I’ve already commented it out, however, new new contributors may experience this issue.

**Comment by guivazcabral** [comment]
The keystore is being injected on the codemagic pipeline, yes. But I don't know if there's a way to change this configs on the pipeline, in order to have it removed from the repository

**Comment by Vanethos** [comment]
We would have to test in a separate branch to check how we can handle it, as soon as possible I will verify with codemagic how we can do that

**Comment by Vanethos** [comment]
@guivazcabral @miguelpruivo 
I suggest the following:

Following the official [Codemagic Guide]()

We can do the following:

```
...
android {
    ...
    defaultConfig { ... }

    signingConfigs {
        release {
            if (System.getenv()["CI"]) { // CI=true is exported by Codemagic
                storeFile file(System.getenv()["FCI_BUILD_DIR"] + "/keystore.jks")
                storePassword System.getenv()["FCI_KEYSTORE_PASSWORD"]
                keyAlias System.getenv()["FCI_KEY_ALIAS"]
                keyPassword System.getenv()["FCI_KEY_PASSWORD"]
            } else {
                storeFile file("/path/to/local/myreleasekey.keystore")
                storePassword "password"
                keyAlias "MyReleaseKey"
                keyPassword "password"
            }
        }
    }

    buildTypes {
        release {
            ...
            signingConfig signingConfigs.release
        }
    }
}
...
```

In this way we can:
- Either create a mock key.properties file
- Remove the `else` statement entirely

If @guivazcabral approves I can try to test this with codemagic.

**Comment by guivazcabral** [comment]
go ahead @Vanethos

**Comment by guivazcabral** [comment]
Solved internally (see closed PR #36)
